src/microbe_stage/editor/PatchMapNode.cs

```diff
@@ -44,6 +44,24 @@ public partial class PatchMapNode : MarginContainer
     [Export]
     private TextureRect glaciationEventIndicator = null!;
 
+    [Export]
+    private TextureRect meteorPlainEventIndicator = null!;
```

Comment:
I think there's way too many static indicator textures...

I think this system should be refactored to dynamically load the textures *as needed* so that each patch map node doesn't all keep references to 8+ textures on them that are rarely used.

Reply:
So would you prefer to have it load textures for ALL events? Like for example have the texture as a separate scene?

Reply:
No, I'd prefer a system where none of the icons exist in the scene, just the container. And then during runtime when the list of active events is updated the code will create and destroy the texture rects as required.